src/test/docker/bionic/Dockerfile

```diff
@@ -5,10 +5,12 @@ LABEL maintainer="Tom Scogland <scogland1@llnl.gov>"
 # Update pkg caches, install latest pkg utils:
 RUN apt-get update \
  && apt-get -qq install -y --no-install-recommends \
-	apt-utils
+	apt-utils \
```

Comment:
Uber minor nit:

Looks like some tabs snuck in but the rest of the file uses space.

Reply:
Good catch! Those may well have been there for quite a while, they are now gone.